.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
+  "trailingComma": "none"
```

Comment:
I would prefer trailingComma was set to "always" or equivalent. It better matches the existing code and works well with auto-formatting tools